package.json

```diff
@@ -109,6 +117,9 @@
     "tslint": "~6.1.0",
```

Comment:
Please remove tslint as it is not needed anymore.

Reply:
updated